extensions/amp-form/amp-form.md

```diff
@@ -196,6 +201,17 @@ You can configure your analytics to send these events as in the example below.
 </amp-analytics>
 ```
 
+`amp-form-submit` is fired before a request is sent. For this event, a set of variables that reflects your form's fields is generated. For an example, consider the following form:
```

Comment:
Some working suggestions:

1. ` is fired before a request is sent` > ` is fired as a request is sent`
2. `reflects your form's fields` > `reflects forms data`

/to @bpaduch for further doc review.